src/utils/IdealGraphVisualizer/Coordinator/src/main/java/com/sun/hotspot/igv/coordinator/OutlineTopComponent.java

```diff
@@ -270,16 +317,268 @@ public void changed(InputGraphProvider lastProvider) {
         }
     }
 
+    private void loadWorkspace() {
+        clear();
+        ((BeanTreeView) this.treeView).setRootVisible(false);
+
+        try {
+            loadGraphDocument(getWorkspaceGraphsPath());
+        } catch (IOException ex) {
+            Exceptions.printStackTrace(ex);
+            return;
+        }
+
+        try {
+            loadStates(getWorkspaceStatesPath());
+        } catch (IOException ex) {
+            Exceptions.printStackTrace(ex);
+        }
+    }
+
     @Override
     public void readExternal(ObjectInput objectInput) throws IOException, ClassNotFoundException {
-        // Not called when user starts application for the first time
         super.readExternal(objectInput);
-        ((BeanTreeView) this.treeView).setRootVisible(false);
+    }
+
+    private String workspacePath;
+
+    private void setWorkspacePath(String path) {
+        changeWorkspaceButton.setText(path);
+        workspacePath = path;
+    }
+
+    static public String WORKSPACE_XML_FILE = "graphs.xml";
+    static public String WORKSPACE_STATE_FILE = "state.igv";
```

Comment:
Let's use blessed modifiers order
```suggestion
    public static String WORKSPACE_XML_FILE = "graphs.xml";
    public static String WORKSPACE_STATE_FILE = "state.igv";
```